thunder/benchmarks/conftest.py

```diff
@@ -0,0 +1,52 @@
+import platform
+import psutil
+from typing import Any
+
+from asvdb import utils, BenchmarkInfo, BenchmarkResult, ASVDb
+
+
+def sanitize_params(benchmark_params: list[tuple[str, Any]]) -> list[tuple[str, Any]]:
+    sane_params = []
+    for k, v in benchmark_params:
+        if k == "executor":
+            sane_params += [(k, str(v).split()[1])]
+        else:
+            sane_params += [(k, v)]
+    return sane_params
+
+
+def pytest_sessionfinish(session, exitstatus):
+    if not hasattr(session.config, "_benchmarksession"):
+        return
+
+    benchmarks = session.config._benchmarksession.benchmarks
+
+    uname = platform.uname()
+    commit_hash, commit_time = utils.getCommitInfo()
+    repo_name, current_branch = utils.getRepoInfo()
+    python_version = platform.python_version()
+    memory_size = str(psutil.virtual_memory().total)
+
+    bench_info = BenchmarkInfo(
+        machineName=uname.machine,
+        osType=f"{uname.system} {uname.release}",
+        pythonVer=python_version,
+        commitHash=commit_hash,
+        commitTime=commit_time,
+        cpuType=uname.processor,
+        arch=uname.machine,
+        ram=memory_size,
+    )
+
+    db = ASVDb(dbDir="./asv", repo=repo_name, branches=[current_branch])
```

Comment:
I'm wondering if it'd do better with an option to specify `dbDir`, e.g., via an env var of `THUNDER_BENCH_DIR`

Reply:
Maybe it could be a pytest command line option, too?